addons/easy_charts/control_charts/domain/chart_axis_domain.gd

```diff
@@ -49,13 +49,13 @@ static func from_values(value_arrays: Array, smooth_domain: bool) -> ChartAxisDo
 
 	var min_max: Dictionary = ECUtilities._find_min_max(value_arrays)
 	if not smooth_domain:
-		domain.lb = min_max.min
+		domain.lb = min(0.0, min_max.min)
 		domain.ub = min_max.max
 		domain.has_decimals = ECUtilities._has_decimals(value_arrays)
 		domain.is_discrete = false
 		domain.fixed = false
 	else:
-		domain.lb = ECUtilities._round_min(min_max.min)
+		domain.lb = ECUtilities._round_min(min(0.0, min_max.min))
```

Comment:
I had to add the `min(0.0, ...)` to ensure that the lowest bar chart y-tick is drawn at zero. Without this, the first tick would be drawn at the lowest value of function y-values. In ase of the bar chart example, that would be at 4.0.